bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/geometry_collector.cpp

```diff
@@ -137,6 +157,8 @@ repo::core::model::RepoNodeSet GeometryCollector::getMeshNodes() {
 			for (const auto &meshMatEntry : meshLayerEntry.second) {
 				if (!meshMatEntry.second.rawVertices.size()) continue;
 
+				auto uvChannels = std::vector<std::vector<repo::lib::RepoVector2D>>();
```

Comment:
I don't think you need the lines at 174
```suggestion
				auto uvChannels = meshMatEntry.second.uvCoords.size() ? std::vector<std::vector<repo::lib::RepoVector2D>>() : {meshMatEntry.second.uvCoords};
```